lib/database/seeders/20230802102102-log-environments.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+    up: async (queryInterface) =>
+        queryInterface.sequelize.transaction((transaction) =>
+            Promise.all([
+                queryInterface.bulkInsert('log_environments', [
+                    {
```

Comment:
Are you sure you need all those seeders? I think one or two is enough. The more you add, the harder it is to maintain

Reply:
I have left enough rows to be able to check that and/or works when filtering